src/containers/search_result.jsx

```diff
@@ -3,15 +3,19 @@ import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 import {
   Table, TableHeader, TableBody, TableHeaderColumn,
-  TableRow, TableRowColumn, Checkbox, SelectField, MenuItem,
+  TableRow, TableRowColumn, Checkbox, FlatButton,
 } from 'material-ui';
 import Visibility from 'material-ui/svg-icons/action/visibility';
 import VisibilityOff from 'material-ui/svg-icons/action/visibility-off';
+// import SortByAlpha from 'material-ui/svg-icons/av/sort-by-alpha';
+import Sort from 'material-ui/svg-icons/content/sort';
+import ArrowDropUp from 'material-ui/svg-icons/navigation/arrow-drop-up';
+import ArrowDropDown from 'material-ui/svg-icons/navigation/arrow-drop-down';
 import Pagination from 'react-ultimate-pagination-material-ui';
 import styled from 'styled-components';
 
 import {
-  SEARCH_RESULTS_COLUMN, SEARCH_SORT_BY,
+  SEARCH_RESULTS_COLUMN,
 } from '../constants';
```

Comment:
If there is only a few importation from a specific package, you can write in just one-line.

Reply:
Finished